packages/datadog_flutter_plugin/test/rum/ddrum_test.dart

```diff
@@ -33,4 +37,89 @@ void main() {
     final other = ContentType.parse('application/octet-stream');
     expect(resourceTypeFromContentType(other), RumResourceType.native);
   });
+
+  test('Tracing sampling rate is clamped to 0..100', () {
+    final lowConfiguration = RumConfiguration(
+      applicationId: 'applicationId',
+      sampleRate: -12.3,
```

Comment:
from the description, `sampleRate` is for the session, so I guess here we should pass `tracingSampleRate`?